### DISCUSSION TITLE
Does Docker (re)Build also Process Modules in Mod Folder?

### DISCUSSION THREAD

**Discussion by Quest79** [azerothcore/azerothcore-wotlk#5624] [Modules]
Does Docker Build also Process Modules in Mod Folder? How do I check if a mod is active?

**Comment by LCGF00** [comment]
I think the easiest and fastest way is to check if the ".conf" configuration files are in the folder etc.

**Reply by Quest79** [reply]
Are in the \env\docker\etc folder?
